board/stm32fx/peripherals.h

```diff
@@ -1,3 +1,17 @@
+void gpio_usb_init(void) {
+  // A11,A12: USB
+  set_gpio_alternate(GPIOA, 11, GPIO_AF10_OTG_FS);
+  set_gpio_alternate(GPIOA, 12, GPIO_AF10_OTG_FS);
+  set_gpio_speed(GPIOA, 11, SPEED_VERY_HIGH);
```

Comment:
why do we set the speed now on the F4? we never did this, correct?

Reply:
if we don't need to set this, let's not

Reply:
it was set before: https://github.com/commaai/panda/blob/3815bf8f41931f957c3cdeeb9cf628109368b110/board/spi_flasher.h#L313